packages/bpk-component-phone-input/readme.md

```diff
@@ -0,0 +1,82 @@
+# bpk-component-phone-input
+
+> Backpack phone input component.
+
+## Installation
+
+```sh
+npm install bpk-component-phone-input --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import BpkPhoneInput from 'bpk-component-phone-input';
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { dialingCode: '44', phoneNumber: '' }
+  }
+
+  onChange = (evt) => {
+    this.setState({ phoneNumber: evt.target.value });
+  }
+
+  onDialingCodeChange = (evt) => {
+    this.setState({ dialingCode: evt.target.value });
+  }
+
+
+  render() {
+    return (
+      <BpkPhoneInput
+        id="phone-input-id"
+        name="Telephone input"
+        placeholder="Telephone number"
+        onChange={this.onChange}
+        onDialingCodeChange={this.onDialingCodeChange}
+        value={this.state.phoneNumber}
+        dialingCode={this.state.dialingCode}
+        dialingCodes={[
+          { code: '44', description: '+44' },
+          { code: '55', description: '+55' },
+        ]}
+        dialingCodeProps={{
+          id: 'dialing-code',
+          name: 'Dialing code',
+          'aria-label': 'Dialing code',
+        }}
+      />
+    )
+  }
+}
+```
+
+## Props
+
+| Property              | PropType                                              | Required   | Default Value    |
+| --------------------- | ----------------------------------------------------- | ---------- | ---------------- |
+| dialingCode           | string                                                | true       | -                |
+| dialingCodeProps      | shape({ id: string, name: string })                   | true       | -                |
+| dialingCodes          | arrayOf(shape({ code: string, description: string })) | true       | -                |
+| id                    | string                                                | true       | -                |
+| name                  | string                                                | true       | -                |
+| onChange              | func                                                  | true       | -                |
+| onDialingCodeChange   | func                                                  | true       | -                |
+| value                 | string                                                | true       | -                |
+| className             | string                                                | false      | null             |
+| containerProps        | object                                                | false      | {}               |
```

Comment:
`container` has [connotations](https://medium.com/@learnreact/container-components-c0e67432e005) in React so we usually use `wrapper` instead.

(Sorry, I should have said this before but I only just noticed)